src/dune_lang/action.ml

```diff
@@ -210,6 +211,9 @@ let decode =
         ; ("ignore-stderr", t >>| fun t -> Ignore (Stderr, t))
         ; ("ignore-outputs", t >>| fun t -> Ignore (Outputs, t))
         ; ("progn", repeat t >>| fun l -> Progn l)
+        ; ( "conc"
+          , Syntax.since Stanza.syntax (3, 7) >>> repeat t >>| fun l -> Conc l
```

Comment:
This PR should be part of the 3.7 millstone or should we bunp to 3.8 ?

Reply:
Probably 3.8.